README.md

```diff
@@ -1,2 +1,3 @@
+//
```

Comment:
This is your first review comment.

Remove this comment to keep the file unchanged. 
Alternatively, make other changes to the file, e.g. add a period (a dot `.`) at the end of the sentence below.

Reply:
Ok, you've removed both markdown files, so you now see how to make changes and push them.